devel/VisualStudio/stdbool.h

```diff
@@ -13,8 +13,8 @@
   include directory as used by VisualStudio. 
 */
 
-#ifndef __STDBOOL_H__
-#define __STDBOOL_H__
+#ifndef ERT_STDBOOL_H
+#define ERT_STDBOOL_H
```

Comment:
Probably not - at the time this written the Visual Studio C compiler was essentially K & R; I think things have improved since then.